agent-inject/agent/agent.go

```diff
@@ -20,6 +20,7 @@ const (
 	DefaultAgentRunAsGroup         = 1000
 	DefaultAgentRunAsSameUser      = false
 	DefaultAgentSetSecurityContext = true
+	defaultServiceAccountPath      = "/var/run/secrets/vault.hashicorp.com/serviceaccount"
```

Comment:
This should be `DefaultServiceAccountPath` to match style of other defaults.

Additionally, is this the correct value?  Typically this is `/var/run/secrets/kubernetes.io/serviceaccount` and I'm not understanding where `vault.hashicorp.com` is coming from.

Reply:
that const isn't used outside the package though so shouldn't it stay private? I think that applies to the other variables too, but just left them as they are

Reply:
the `defaultServiceAccountPath` would be the default for using projected service account tokens, as the ' normal' service account token is already being searched for - we can change this, but I think it'd make sense to have one that isn't the same as the normal path

Reply:
@dippynark Yes these aren't used outside of the package, but it should still follow style present with the other vars.

Reply:
I missed the fallback search, so this should be OK.